.github/environments/values.dev.yaml

```diff
@@ -6,6 +6,7 @@ config:
     FRONTEND_COMS_APIPATH: https://coms-dev-master.apps.silver.devops.gov.bc.ca/api/v1
     FRONTEND_OIDC_AUTHORITY: https://dev.loginproxy.gov.bc.ca/auth/realms/standard
     FRONTEND_OIDC_CLIENTID: bc-box-4555
+    FRONTEND_NOTIFICATION_HOME_BANNER: This is the DEV environment of BCBox. Uploaded files may not persist and may be deleted from the COMS database as we continue development. Please do not upload personal or private information. Thank you for your understanding as we work to improve BCBox
```

Comment:
Please have this be inserted in alphabetical order. In this case, it should come before the OIDC keys.

Reply:
Fixed